rmgpy/cantherm/molpro.py

```diff
@@ -260,58 +260,89 @@ def loadConformer(self, symmetry=None, spinMultiplicity=0, opticalIsomers=1, sym
+                    f12a = True
+                    # MRCI could also have a vdz/vtz basis, so don't break
+                elif any(high_basis in line.lower() for high_basis in ['vqz', 'v5z', 'v6z', 'v7z', 'v8z']):
+                    f12b = True
+                    # MRCI could also have a v(4+)z basis, so don't break
+            elif 'mrci' in line.lower():
+                mrci = True
+                f12a, f12b = False, False
                 break
+            elif 'point group' in line.lower():
+                # we expect to already know the method by this point, so break if possible, but don't throw an error yet
+                if any([mrci, f12a, f12b]):
+                    break
             line=f.readline()
-        else: raise Exception('Could not find basis set in Molpro File')
-        #search for energy
-        E0=None
+        else:
+            raise ValueError('Could not determine type of calculation. Currently, CCSD(T)-F12a, CCSD(T)-F12b,'
+                             ' MRCI, MRCI+Davidson are supported')
+        # search for energy
+        E0 = None
         if f12a:
-            while line!='':
+            while line != '':
                 if ('RHF-UCCSD(T)-F12a energy' in line
                     or 'RHF-RCCSD(T)-F12a energy' in line
-                    or 'CCSD(T)-F12a total energy  ' in line):
-                    E0=float(line.split()[-1])
-                    break
+                        or 'CCSD(T)-F12a total energy  ' in line):
+                        E0 = float(line.split()[-1])
+                        break
                 if 'Electronic Energy at 0' in line:
-                    E0=float(line.split()[-2])
+                    E0 = float(line.split()[-2])
                     break
                 line=f.readline()
-        else:
-            while line!='':
+        elif f12b:
+            while line != '':
                 if ('RHF-UCCSD(T)-F12b energy' in line
                     or 'RHF-RCCSD(T)-F12b energy' in line
-                    or 'CCSD(T)-F12b total energy  ' in line):
-                    E0=float(line.split()[-1])
-                    break
+                        or 'CCSD(T)-F12b total energy  ' in line):
+                        E0 = float(line.split()[-1])
+                        break
                 if 'Electronic Energy at 0' in line:
-                    E0=float(line.split()[-2])
+                    E0 = float(line.split()[-2])
                     break
                 line=f.readline()
-
+        elif mrci and 'mrci+davidson' in model_chemistry.lower():
+            while line != '':
+                if '(Davidson, relaxed reference)' in line:
+                    E0 = float(line.split()[3])
+                    break
+                line=f.readline()
+        elif mrci and 'mrci' in model_chemistry.lower():  # 'mrci+davidson' is not in the model chemistry
+            while line != '':
+                if 'MRCI            MULTI           HF-SCF' in line:
```

Comment:
Will these exact number of spaces be consistent across different versions of Molpro? Perhaps it would be more prudent to just check if `MRCI`, `MULTI`, and `HF-SCF` are all in the line.
```suggestion
                if all(w in line for w in ('MRCI', 'MULTI', 'HF-SCF')):
```